modules/microshift-nw-config-route-admission.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-configuring-route-admission_{context}"]
+= Configuring route admission policy
+
+By default, {microshift-short} prevents routes in multiple namespaces from using the same hostname. To accommodate different use cases, you can use the configuration option below to disable the `namespaceOwnership` check. This allows routes to claim different paths of the same hostname across namespaces.
```

Comment:
```suggestion
By default, {microshift-short} allows routes in multiple namespaces to use the same hostname. To prevent routes in different namespaces from claiming the same hostname, you can configure the value. 
```
It defaults to allowing. See https://github.com/openshift/openshift-docs/pull/76100/files#diff-5968debd1075f10b57e348a65f0cedec425dc131c575c44f417d5ebe0a272841R76 per QE comments.

Reply:
This was my bad when reading it, by default it *allows* routes. This is contrary to what OpenShift does, which defaults to Strict.

Reply:
So please make all of the changes I suggested here, @aravipra.

Reply:
done